server/controller/src/main/java/ai/starwhale/mlops/configuration/security/ProjectNameExtractorDataStoreMixed.java

```diff
@@ -92,7 +101,12 @@ public ProjectNameExtractorDataStoreMixed(
             ModelDao modelDao,
             DatasetDao datasetDao,
             RuntimeDao runtimeDao,
-            ReportDao reportDao, TemplateDao templateDao) {
+            ReportDao reportDao,
+            TemplateDao templateDao,
+            ModelVersionMapper modelVersionMapper,
```

Comment:
Does it use xxDao.getXXVersion() enough? so that the xxMapper can be removed.

Reply:
Fixed